docs/introduction/javascript-events-dom-apis.md

```diff
@@ -502,6 +502,41 @@ entityEl.addEventListener('physicscollided', collisionHandler);
 entityEl.removeEventListener('physicscollided', collisionHandler);
 ```
 
+### Binding Event Listeners
+
+By default, event listeners do not have access to the component's `this`.  Instead, `this` will be bound to `window`, so you won't see errors if you reference `this` inside an event Listener, but you are likely to get unexpected results.
```

Comment:
It's not specific to event listeners but applies to any function. We should probably start with that. Maybe:

By default, [Javascript execution context rules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this) binds `this` to the global context (`window`) for any independent function. To access the component object from an event listener there are several options:

Reply:
Updated with your suggestion.